python/lsst/ap/association/packageAlerts.py

```diff
@@ -267,28 +272,36 @@ def run(self,
         sciencePsf = self._computePsf(calexp, calexp.psf.getAveragePosition())
         templatePsf = self._computePsf(template, template.psf.getAveragePosition())
 
+        if ssSrc:
+            ssSrc.add_index('diaSourceId')
+
         n_sources = len(diaSourceCat)
         self.log.info("Packaging alerts for %d DiaSources.", n_sources)
         # Log every 10 seconds as proof of liveness.
         loop_logger = lsst.utils.logging.PeriodicLogger(self.log, interval=10.0)
 
         for srcIndex, diaSource in diaSourceCat.iterrows():
             loop_logger.log("%s/%s sources have been packaged.", len(alerts), n_sources)
-            # Get all diaSources for the associated diaObject.
-            # TODO: DM-31992 skip DiaSources associated with Solar System
-            #       Objects for now.
+
+            alertId = diaSource["diaSourceId"]
             if srcIndex[0] == 0:
```

Comment:
I believe the diaObjectId is 0 because it defaults that way on construction, and we never change it.  To actually set it to NULL I think we'd have to do it in https://github.com/lsst/pipe_tasks/blob/main/python/lsst/pipe/tasks/ssoAssociation.py.

(Probably ssObjectId is also 0 for non-SSO diaSources as well...)